accounts/ethkey/cli/src/main.rs

```diff
@@ -267,16 +317,13 @@ fn execute<S, I>(command: I) -> Result<String, Error> where I: IntoIterator<Item
 		let (phrase, keypair) = in_threads(move || {
 			let mut it = brain_recover::PhrasesIterator::from_known_phrase(&known_phrase, BRAIN_WORDS);
 			move || {
-				let mut i = 0;
-				while let Some(phrase) = it.next() {
-					i += 1;
-
+				for (i, phrase) in (&mut it).enumerate() {
 					let keypair = Brain::new(phrase.clone()).generate().unwrap();
 					if keypair.address() == address {
 						return Ok(Some((phrase, keypair)))
 					}
 
-					if i >= 1024 {
+					if i > 1024 {
```

Comment:
this breaks the current functionality, i.e, before `1024` was enough to breakout out of the loop with this change it is `1025`

please revert or explainn

Reply:
Before the change the loop effectively started with 1 because of how increment was structured. I made the change to balance the fix in the beginning of the scope.

Reply:
right, missed that.